### DISCUSSION TITLE
Bluetooth persist pairing

### DISCUSSION THREAD

**Discussion by boatbodger** [earlephilhower/arduino-pico#1923] [Q&A]
On a Pico-W project, I'm using the SerialBT library to create the ability to configure my Arduino app, and using my Android phone running a Bluetooth terminal program (Serial Bluetooth Terminal 1.47).
This works, but I find that each time I restart the Pico-W, I have to re-pair with the Android - which is tedious.
Is there a way of avoiding the need for re-pairing each time?  I have no real need for security - the target app is simply an LED pattern generator!

**Comment by nWestie** [comment]
Any luck with this? I ran into the same issue when trying to set up with android, and haven't had much luck finding answers

**Reply by boatbodger** [reply]
In my case, I was using the Pico-W's for a home project, so now they are set up I've not needed to fire up the Bluetooth connection again.  Your comment is the only one my post has received, so sorry, can't offer any help.
